services/app-api/src/emailer/emails/newContractStateEmail.test.ts

```diff
@@ -154,27 +154,28 @@ test('includes mcog, rate, and team email addresses', async () => {
         throw template
     }
 
+    // Note when anchor links are generated there is an extra space for this test
```

Comment:
As I'm investigating this, it doesn't seem like this is intentional so I might modify the helper that is causing the extra space to appear.

Please let me know if you think we shouldn't be making this change.